CHANGELOG.md

```diff
@@ -4706,8 +4759,8 @@ Full Changelog: [v3.0.10...v3.1.0-rc1](https://github.com/argoproj/argo-workflow
 * Riccardo Piccoli
 * Roi Kramer
 * Rush Tehrani
-* Saïfane FARFAR
 * Saravanan Balasubramanian
+* Saïfane FARFAR
```

Comment:
I have no idea why the ordering changed in a few random author names 🤷  possibly a different `git` or Linux version's unicode sorting? Since these are all changes to characters that are from languages other than American English (this one is `Saïfane FARFAR` and the others are `张志强`)